pages/api/gifts/[id].ts

```diff
@@ -49,17 +49,19 @@ async function handleGET({ res, queryId }: HandlerParams) {
 
 async function handlePATCH({ req, res, queryId }: HandlerParams) {
   const patchRequest = await axios.patch(`${baseURL}/${queryId}`, req.body);
-  return res.status(patchRequest.status).json(req.body);
+  return res.status(patchRequest.status).json(patchRequest.data as Gift);
 }
 
 async function handlePUT({ req, res, queryId }: HandlerParams) {
   const putRequest = await axios.put(`${baseURL}/${queryId}`, req.body);
-  return res.status(putRequest.status).json(req.body);
+  return res.status(putRequest.status).json(putRequest.data as Gift);
 }
 
-async function handleDELETE({ req, res, queryId }: HandlerParams) {
+async function handleDELETE({ res, queryId }: HandlerParams) {
   const deleteRequest = await axios.delete(`${baseURL}/${queryId}`);
-  return res.status(deleteRequest.status).json(req.body);
+  return res
+    .status(deleteRequest.status)
+    .send('Lahja poistettu onnistuneesti!');
 }
```

Comment:
Laitoin tuohon oman tekstin, kun lahja on poistettu, lienee melko turha.

Voisin laittaa kaikki virheet, return-statementistä lähetettävät tiedot yms englanniksi. Kääntää ne vain suomeksi sitten, kun tekee toastin.

Esim. että tuo `'Lahja poistettu onnistuneesti!'` olisikin `'Gift deleted succesfully'` yms

Reply:
Jep status-tekstit joita ei suoraan näytetä käyttäjälle kannattaa kirjoittaa englanniksi. Tähän en kuitenkaan laittaisi status-tekstiä, sillä yleensä DELETE-request ei palauta mitään. Se että lahjan poisto onnistui kerrotaan status-koodilla (200). Jos poisto ei jostain syystä onnistu palautetaan esim. koodi 400 tai 500.

Reply:
Joo, näin ajattelinkin 👍